presto-main/src/test/java/com/facebook/presto/operator/scalar/AbstractTestFunctions.java

```diff
@@ -198,6 +199,23 @@ protected void assertInvalidCast(String projection, String message)
         }
     }
 
+    protected void assertNotSupported(String projection, Type expectedType, String message)
```

Comment:
Don't take `expectedType` here. We shouldn't need a type for an invalid function.